lib/Connector/AlphaVantageConnector.php

```diff
@@ -118,6 +121,83 @@ public function onDataRequest(WidgetDataRequestEvent $event)
         }
     }
 
+    public function onWidgetEditOption(WidgetEditOptionRequestEvent $event)
+    {
+        $this->getLogger()->debug('onWidgetEditOption');
+
+        // Pull the widget we're working with.
+        $widget = $event->getWidget();
+        if ($widget === null) {
+            throw new NotFoundException();
+        }
+
+        // We handle the twitter widget and the property with id="type"
+        if ($widget->type === 'stocks' && $event->getPropertyId() === 'searchSymbol') {
+
```

Comment:
It would be good to make sure `phpcs` checking is working in your IDE, because I think this empty line will flag up here.